crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -101,5 +111,19 @@ def failed?
     def upgrade_script_path
       Rails.root.join("..", "bin", "upgrade_admin_server.sh")
     end
+
+    def addon_installed?(addon)
+      case addon
+      when "ceph"
+        CephService
+      when "ha"
+        PacemakerService
+      else
+        return false
+      end
+      true
```

Comment:
no actually it would return `PacemakerService` or `CephService` which probably evaluates to `true` in an if context, but it wouldn't be correct. and this is more explicit